WordPress/src/test/java/org/wordpress/android/ui/posts/editor/media/AddExistingMediaToPostUseCaseTest.kt

```diff
@@ -0,0 +1,121 @@
+package org.wordpress.android.ui.posts.editor.media
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.anyOrNull
+import com.nhaarman.mockitokotlin2.argumentCaptor
+import com.nhaarman.mockitokotlin2.doAnswer
+import com.nhaarman.mockitokotlin2.inOrder
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.times
+import com.nhaarman.mockitokotlin2.verify
+import kotlinx.coroutines.InternalCoroutinesApi
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+import org.wordpress.android.fluxc.model.MediaModel
+import org.wordpress.android.test
+import org.wordpress.android.ui.posts.editor.EditorTracker
+
+@RunWith(MockitoJUnitRunner::class)
+@UseExperimental(InternalCoroutinesApi::class)
+class AddExistingMediaToPostUseCaseTest {
+    @Rule
+    @JvmField val rule = InstantTaskExecutorRule()
```

Comment:
NIT: I've created the `BaseUnitTest` class so I can avoid this but I'm fine with both approaches :-)

Reply:
Fixed in aabc40875ab89c50aeb3202564b1348d65de266f